ansible/roles_ocp_workloads/ocp4_workload_example/tasks/remove_workload.yml

```diff
@@ -2,8 +2,6 @@
 # Implement your workload removal tasks here
 # ------------------------------------------
 
-
```

Comment:
There's a reason I left the empty lines..... I wanted to make sure it was obvious where to add "things".